Documentation/kubernetes.md

```diff
@@ -36,6 +36,105 @@ kubectl create -f rook-operator.yaml
 # verify the rook-operator and rook-agents pods are in the `Running` state before proceeding
 kubectl -n rook-system get pod
 ```
+## RBAC for PodSecurityPolicies
+
+If you have activated the [PodSecurityPolicy Admission Controller](https://kubernetes.io/docs/admin/admission-controllers/#podsecuritypolicy) and thus are
+using [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/), you will require additional `(Cluster)RoleBindings` 
+for the different `ServiceAccounts` Rook uses to start the Rook Storage Pods.
+ 
+**Note**: You don not have to perform these steps if you do not have the `PodSecurityPolicy` Admission Controller activated!  
+
+##### PodSecurityPolicy 
+
+You will probably have a `PodSecurityPolicy` for elevated access in place, but you need one privileged Pod execution. Here is an example:
```

Comment:
"[...]but you need one privileged Pod execution" do you mean "you need one `PodSecurityPolicy` that allows privileged Pod execution"